### PR TITLE
Prepare changelog

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Related to #3098

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,45 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
+
+### BPMN
+
+* `FEAT`: show diagram errors on canvas ([#3118](https://github.com/camunda/camunda-modeler/pull/3118))
+* `FEAT`: add support for extension properties in Camunda Platform 8.1 ([#731](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/731))
+* `FEAT`: show error on duplicate task header keys ([#4](https://github.com/camunda/linting/issues/4))
+* `FIX`: only claim existing element IDs ([#3086](https://github.com/camunda/camunda-modeler/issues/3085))
+* `FIX`: move labels when collapsing sub processes ([#1695](https://github.com/bpmn-io/bpmn-js/issues/1695))
+* `FIX`: render sequence flows always on top ([#1716](https://github.com/bpmn-io/bpmn-js/issues/1716))
+* `FIX`: show FEEL syntax errors in properties panel ([#173](https://github.com/bpmn-io/properties-panel/pull/173))
```
Comment (marstamm) [inline CHANGELOG.md:31]:
```suggestion
* `FIX`: show FEEL syntax errors in properties panel ([#173](https://github.com/bpmn-io/properties-panel/pull/173))
* `FIX`: focus expanded FEEL editor on click ([#3072](https://github.com/camunda/camunda-modeler/issues/3072))
```

Comment (marstamm) [reply CHANGELOG.md:31]: Could not find #3072 in the change log, but let me know if I just missed it

Comment (pinussilvestrus) [reply CHANGELOG.md:31]: I simply missed that one, was also not in the milestone. I will add, thanks 👍

Comment (pinussilvestrus) [reply CHANGELOG.md:31]: done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,46 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
+* `FIX`: remove user path from Mixpanel tracking ([#3105](https://github.com/camunda/camunda-modeler/pull/3105))
+* `DEPS`: bump @camunda/linting to v0.5.0 ([#3118](https://github.com/camunda/camunda-modeler/pull/3118))
+* `DEPS`: bump codemirror to v6.0.1 ([#3103](https://github.com/camunda/camunda-modeler/pull/3103))
+* `DEPS`: bump diagram-js to v8.9.0 ([#3113](https://github.com/camunda/camunda-modeler/pull/3113))
+* `DEPS`: bump @bpmn-io/properties-panel to v0.20.1 ([#3120](https://github.com/camunda/camunda-modeler/pull/3120))
+
+### BPMN
+
+* `FEAT`: show diagram errors on canvas ([#3118](https://github.com/camunda/camunda-modeler/pull/3118))
+* `FEAT`: add support for extension properties in Camunda Platform 8.1 ([#731](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/731))
```
Comment (philippfromme) [inline CHANGELOG.md:26]: We could add the lint rule as well: https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/43

Comment (pinussilvestrus) [reply CHANGELOG.md:26]: Yes, thanks, added!

> * `FEAT`: allow `zeebe:Properties` only in Camunda Platform 8.1 or higher ([#43](https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/43))

**Comment by pinussilvestrus** [comment]
@marstamm @philippfromme can we merge this one?

**Pull request merged**